.travis.yml

```diff
@@ -1,23 +1,39 @@
 language: android
+dist: trusty
```

Comment:
This is a 5 year old release. If this is needed, please add a comment.

Reply:
`Android builds are only supported on our Trusty image at this time hence you’ll need to explicitely specify dist: trusty in your .travis.yml file.`

https://docs.travis-ci.com/user/languages/android/